tutorials/segmentation.ipynb

```diff
@@ -0,0 +1,239 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "kidneys = segmentation.get_kidneys()\n",
+    "left_kidney = segmentation.get_left_kidney()\n",
+    "right_kidney = segmentation.get_right_kidney()\n",
+    "\n",
+    "# Print kidney volumes.\n",
+    "print(f'TKV = {segmentation.get_tkv():.2f} ml\\n'\n",
+    "      f'LKV = {segmentation.get_lkv():.2f} ml\\n'\n",
+    "      f'RKV = {segmentation.get_rkv():.2f} ml')\n",
+    "\n",
+    "# Alternatively the volumes can be retrieved as a dictionary or saved as a csv.\n",
+    "volumes = segmentation.get_volumes()\n",
+    "segmentation.save_volumes_csv(os.path.join(OUTPUT_DIR, 'volumes.csv'))\n",
+    "\n",
+    "# Display individual kidney masks.\n",
+    "fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(15, 4))\n",
+    "ax1.imshow(kidneys[:, :, 7].T, origin='lower')\n",
+    "ax1.set_title(f\"TKV = {volumes['total']:.0f} ml\")\n",
+    "ax1.axis(False)\n",
+    "ax2.imshow(left_kidney[:, :, 7].T, origin='lower')\n",
+    "ax2.set_title(f\"LKV = {volumes['left']:.0f} ml\")\n",
+    "ax2.axis(False)\n",
+    "ax3.imshow(right_kidney[:, :, 7].T, origin='lower')\n",
+    "ax3.set_title(f\"RKV = {volumes['right']:.0f} ml\")\n",
+    "ax3.axis(False)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Finally, we can save the masks and niftis. Here all possible output options are explicitly listed, the same result could be achieved by simply using `maps='all'`."
```

Comment:
Finally, we can save the masks and NIfTIs. Here all possible output options are explicitly listed, but the same result could be achieved by simply using `maps='all'`."